src/ui/RealmBrowser/Content/Table/types/ListCell.tsx

```diff
@@ -67,13 +67,14 @@ const displayValue = (
     if (list.length > 0) {
       // Check if this type of object has a primary key
       const firstObject: Realm.Object = list[0];
+      // Note: (QuickFix) Since realm@10.2.0 we started receiving realm objects without `objectSchema()` defined.
+      if (!firstObject.objectSchema) return;
```

Comment:
I think we need to locate the root cause of this.

Reply:
100% agreed

Reply:
Ok I investigated, and - I goofed...
This was fixed by other edits in this PR (it was not yet detecting a `Realm.List<UUID>` as primitives... and treated them as a list of `Realm.Object`).

So this is a non-issue.
I rolled back the quickfix.